src/FormFeedback.js

```diff
@@ -8,26 +8,30 @@ const propTypes = {
   tag: PropTypes.string,
   className: PropTypes.string,
   cssModule: PropTypes.object,
-  valid: PropTypes.bool
+  valid: PropTypes.bool,
+  validMode: PropTypes.oneOf(['feedback', 'tooltip'])
```

Comment:
I would prefer a prop named `tooltip` so the usage would be:
```jsx
<FormFeedback>Oh noes! that name is already taken</FormFeedback>
vs
<FormFeedback tooltip>Oh noes! that name is already taken</FormFeedback>
```

Reply:
I was doing that at the same time you were commenting it ;)